GroupIS1024/Pages/Index.cshtml

```diff
@@ -37,10 +38,21 @@
         .intro p {
             text-align: center;
         }
-    </style>
 
+        h2 {
+            margin-bottom: 30px;
+        }
 
+        h3 {
+            text-align: center;
+        }
 
+        .gmap_canvas {
+            position: center;
+            width: 100%;
+            height: 100%;
+        }
+    </style>
```

Comment:
For XHTML 1.0 Strict compatibility, move all styling to external css.  Embedded and inline styles mix content with look and feel, where HTML should only be used to structure content.